rust/bin/agent/src/handler/insert.rs

```diff
@@ -166,9 +188,40 @@ impl insert_server::Insert for super::Agent {
     #[doc = " A method to add new multiple vectors by bidirectional streaming.\n"]
     async fn stream_insert(
         &self,
-        _request: tonic::Request<tonic::Streaming<insert::Request>>,
+        request: tonic::Request<tonic::Streaming<insert::Request>>,
     ) -> std::result::Result<tonic::Response<Self::StreamInsertStream>, tonic::Status> {
-        todo!()
+        info!("Received stream insert request from {:?}", request.remote_addr());
+
+        let hostname = cargo::util::hostname()?;
+        let _domain = hostname.to_str().unwrap();
+        let _resource_type = self.resource_type.clone() + "/qbg.StreamInsert";
+        let _resource_name = format!("{}: {}({})", self.api_name, self.name, self.ip);
+
+        let s = self.s.clone();
+        let resource_type = self.resource_type.clone();
+        let name = self.name.clone();
+        let ip = self.ip.clone();
+        let api_name = self.api_name.clone();
+
+        let process_fn = move |req: insert::Request| {
+            let s = s.clone();
+            let resource_type = resource_type.clone();
+            let name = name.clone();
+            let ip = ip.clone();
+            let api_name = api_name.clone();
+            async move {
+                match insert(s, &resource_type, &api_name, &name, &ip, &req).await {
+                    Ok(response) => {
+                        Ok(object::StreamLocation {
+                            payload: Some(object::stream_location::Payload::Location(response)),
+                        })
+                    }
+                    Err(status) => Err(status),
+                }
+            }
+        };
+
+        bidirectional_stream(request, 10, process_fn).await
```

Comment:
Does `10` mean a temporary number?
We should leave `todo!()` above this line to set value (e.g. `concurrency`) instead of it when we implement config binding logic.